firmware/application/file_path.cpp

```diff
@@ -22,28 +23,104 @@
 #include "file_path.hpp"
 #include "file.hpp"
 
-const std::filesystem::path adsb_dir = u"ADSB";
-const std::filesystem::path ais_dir = u"AIS";
-const std::filesystem::path apps_dir = u"APPS";
-const std::filesystem::path aprs_dir = u"APRS";
-const std::filesystem::path audio_dir = u"AUDIO";
-const std::filesystem::path blerx_dir = u"BLERX";
-const std::filesystem::path bletx_dir = u"BLETX";
-const std::filesystem::path captures_dir = u"CAPTURES";
-const std::filesystem::path debug_dir = u"DEBUG";
-const std::filesystem::path firmware_dir = u"FIRMWARE";
-const std::filesystem::path freqman_dir = u"FREQMAN";
-const std::filesystem::path gps_dir = u"GPS";
-const std::filesystem::path logs_dir = u"LOGS";
-const std::filesystem::path looking_glass_dir = u"LOOKINGGLASS";
-const std::filesystem::path playlist_dir = u"PLAYLIST";
-const std::filesystem::path remotes_dir = u"REMOTES";
-const std::filesystem::path repeat_rec_path = u"CAPTURES";
-const std::filesystem::path samples_dir = u"SAMPLES";
-const std::filesystem::path screenshots_dir = u"SCREENSHOTS";
-const std::filesystem::path settings_dir = u"SETTINGS";
-const std::filesystem::path spectrum_dir = u"SPECTRUM";
-const std::filesystem::path splash_dir = u"SPLASH";
-const std::filesystem::path sstv_dir = u"SSTV";
-const std::filesystem::path wav_dir = u"WAV";
-const std::filesystem::path whipcalc_dir = u"WHIPCALC";
+const std::filesystem::path adsb_dir = u"SYS/ADSB";  // this dir no need to seperate profiles since all are resources
+
+const std::filesystem::path ais_dir = u"SYS/AIS";  // this dir no need to seperate profiles since all are resources
+
+const std::filesystem::path apps_dir = u"APPS";  // currently in root
+
+// APRS
+const std::filesystem::path aprs_dir = u"APRS";  // fake path
+const std::filesystem::path aprs_dir_user = u"/APRS";
+const std::filesystem::path aprs_dir_resources = u"SYS/APRS";
+
+// AUDIO
+const std::filesystem::path audio_dir = u"AUDIO";  // fake path
+const std::filesystem::path audio_dir_user = u"/AUDIO";
+const std::filesystem::path audio_dir_resources = u"SYS/AUDIO";
+
+// BLERX
+const std::filesystem::path blerx_dir = u"BLERX";  // fake path
+const std::filesystem::path blerx_dir_user = u"/BLERX";
+const std::filesystem::path blerx_dir_resources = u"SYS/BLERX";
+
+// BLETX
+const std::filesystem::path bletx_dir = u"BLETX";  // fake path
+const std::filesystem::path bletx_dir_user = u"/BLETX";
+const std::filesystem::path bletx_dir_resources = u"SYS/BLETX";
```

Comment:
All the duplicated folder names in this file still bother me.  Can't you add little functions to add the user or system path prefix, something like:

std::filesystem::path system_path(std::filesystem::path base_path) {
    return u"SYS/" + base_path;
}
std::filesystem::path user_path(std::filesystem::path base_path) {
    return u"/" + base_path;
}

Reply:
(Anymore in this PR)